src/ir/branch-utils.h

```diff
@@ -94,6 +101,10 @@ void operateOnScopeNameUsesAndSentValues(Expression* expr, T func) {
       func(name, sw->value);
     } else if (auto* br = expr->dynCast<BrOn>()) {
       func(name, br->ref);
+    } else if (auto* tt = expr->dynCast<TryTable>()) {
+      // The values are supplied by throwing instructions, so we are unable to
+      // know what they will be here.
+      func(name, nullptr);
```

Comment:
Before, `nullptr` meant "no value" but now it can also be "there is a value, but it is not known statically", which worried me initially but after reading the uses in  `Heap2Local` and `possible-contents.cpp` that looks ok. IIUC these branches are from throws which we can't identify statically (and might even be in another function). Please update the comment on line 92, though, to document this API change.

Reply:
Done